src/core/api/payment_manager_api/admin-panel/_base_policy.xml.tpl

```diff
@@ -11,6 +11,19 @@
           <method>HEAD</method>
         </allowed-methods>
       </cors>
+      <choose>
+      <!-- TODO Enable after ticket resolution -->
+      <!-- <when condition="@(context.Request.Headers.GetValueOrDefault("X-Environment", "").Equals("uat"))">
+        <check-header name="X-Forwarded-For" failed-check-httpcode="403" failed-check-error-message="Unauthorized" ignore-case="true">
+          <value>${allowed_ip}</value>
+        </check-header>
+      </when> -->
+      <when condition="@(context.Request.Headers.GetValueOrDefault("X-Environment", "").Equals("prod"))">
```

Comment:
@pasqualedevita why only prod?

Reply:
Policy is disabled in UAT until Microsoft ticket response